runtime/particle-execution-host.js

```diff
@@ -88,9 +88,9 @@ export class ParticleExecutionHost {
 
     this._apiPort.onGetBackingStore = async ({callback, type, storageKey}) => {
       if (!storageKey) {
-        storageKey = this._arc._storageProviderFactory.baseStorageKey(type, this._arc._storageKey || 'volatile');
```

Comment:
I'd like to codify it but I'm not sure we can given that we tend to use underscores for hidden members with public getters:

```
class Foo {
  private _gettable: string;
  get gettable() { return _gettable; }
}
```

We can't convert these to `readonly` because the semantic is usually that they should be modifiable but only by the instance that owns them.